lib/abr/simple_abr_manager.js

```diff
@@ -128,6 +136,24 @@ shaka.abr.SimpleAbrManager = class {
     // can start using bandwidth estimates right away after init() is called.
   }
 
+  /**
+   * @override
+   * @export
+   */
+  destroy() {
```

Comment:
Since this isn't actually asynchronous, you should implement `shaka.util.IReleasable` instead.